src/main/scala/org/dia/algorithms/mcc/GTGRunner.scala

```diff
@@ -336,6 +336,15 @@ class GTGRunner(val masterURL: String,
     processEdges(MCCEdgeList, MCCNodeMap)
 
     /**
+     * Generate the netcdfs
+     */
+    val broadcastedNodeMap = sc.sparkContext.broadcast(MCCNodeMap)
+    val edgeTupleList = sc.sparkContext.parallelize(MCCEdgeList.toList).map({ x =>
+      (s"${x.srcNode.frameNum},${x.srcNode.cloudElemNum}",
+      s"${x.destNode.frameNum},${x.destNode.cloudElemNum}") })
+    MCSUtils.get_all_node_data(edgeTupleList.collect().toArray, broadcastedNodeMap.value, lat, lon, false)
```

Comment:
Lets avoid passing function parameters with chained function calls
Instead of 
MCSUtils.get_all_node_data(edgeTupleRDD.collect() ...,                 
do
val edgeTuples = edgeTupleRDD.collect
MCSUtils.get_all_node_data(edgeTuples....

I find this helps readability, and aids in debugging (especially when using step-through debuggers).